app/lib/efile/ny/it201.rb

```diff
@@ -89,7 +90,7 @@ def calculate
         set_line(:IT201_LINE_67, -> { @lines[:IT214_LINE_33].value })
         set_line(:IT201_LINE_69, :calculate_line_69)
         set_line(:IT201_LINE_69A, :calculate_line_69a)
-        set_line(:IT201_LINE_70, -> { @lines[:IT215_LINE_27].value})
+        set_line(:IT201_LINE_70, -> { @lines[:IT215_LINE_27] ? @lines[:IT215_LINE_27].value : 0})
```

Comment:
this was breaking if nyc_full_year_resident was false cuz line 27 is only calculated if it's true